### ISSUE TITLE
Can I set how many pixels close to the border the cursor changes to two arrows?

### ISSUE THREAD

**Issue by jack6th** [mpv-player/mpv#8289]
- mpv 0.32 on Windows 10

Open a video with `mpv --border=no --geometry=320`   
When I move the cursor close to the border (not exact at the border) the cursor changes to two arrows.   
I cannot click _ ◻️ x at the right corner and the volume, fullscreen buttons below unless I make the window larger.   

Can I set how many pixels close to the border the cursor changes to two arrows?

**Comment by bitingsock** [comment]
I think this is a window manager feature.

**Comment by avih** [comment]
> I think this is more of a window manager feature.

Not 100% sure of that. Seems like we do a custom calculation of the areas which change the cursor to resize when borderless - https://github.com/mpv-player/mpv/blob/master/video/out/w32_common.c#L175

@rossy might know more.

**Comment by rossy** [comment]
Yeah, that calculation was my attempt at copying the size and shape of the normal Windows sizing handles. So I guess we could make them configurable. I don't know if that's the right solution though. Maybe we should just make sure the OSC can't get that small? (#8294)

**Comment by bitingsock** [comment]
what about an option to just turn off the corners and/or sides?

**Comment by AjvarXX** [comment]
There should be option to set (or built-in) minimum size/height of a bar (both upper and lower).
Here I asked about the upper one https://github.com/mpv-player/mpv/issues/8294

**Comment by AjvarXX** [comment]
Sorry... posted in the wrong topic. But can't delete posts for some reason.
